src/IECoreNuke/FromNukeCameraConverter.cpp

```diff
@@ -113,22 +81,17 @@ IECore::ObjectPtr FromNukeCameraConverter::doConversion( IECore::ConstCompoundOb
 			// fall through to orthographic code
 		case DD::Image::CameraOp::LENS_ORTHOGRAPHIC :
 		{
-			result->parameters()["orthographic"] = new IECore::StringData( "orthographic" );
+			result->setProjection( "orthographic" );
 			break;
 		}
 	}
 
-	// screen window. nuke fits the field of view horizontally to the width of the image, but we also need
-	// to take into account the window translate and scale.
-	float oneOverAspectRatio = (float)resolution.y / (float)resolution.x;
-	float screenWindowHalfWidth = m_camera->projection_mode() == DD::Image::CameraOp::LENS_PERSPECTIVE ? 1.0f : 0.5f * filmWidth / m_camera->focal_length();
-	Box2f screenWindow( V2f( -screenWindowHalfWidth, -screenWindowHalfWidth * oneOverAspectRatio ), V2f( screenWindowHalfWidth, screenWindowHalfWidth * oneOverAspectRatio ) );
-	V2f screenWindowTranslate = IECore::convert<Imath::V2f>( m_camera->win_translate() );
+	// TODO - I haven't tested any of this - I'm not sure how to use it, because it doesn't appear to
+	// be used anywhere.  Why does it even exist?
```

Comment:
I think it was used by our short-lived "3delight for Nuke" integration. Unless anyone has an alternative use case, I propose we remove it.

Reply:
It would be necessary if we wanted to publish cameras from Nuke (or if we ever get CaribouNuke working). But if its not working currently and not tested, its probably not a big deal to start over when that time comes...

Reply:
Speaking with Andrew about it, he seemed open to the idea of keeping this around because we do have tickets to work on Nuke integration.  While I haven't tested this, I suspect it's mostly working, and would provide a reasonable basis for that work.

Reply:
My preference is to have less code to maintain, but it's your call...